blueprints/jsonapi-adapter/files/__root__/__path__/__name__.js

```diff
@@ -1,17 +1,5 @@
 <%= importStatement %>
-import config from '../config/environment';
 
 export default <%= baseClass %>.extend({
   type: '<%= entity %>',
-
-  url: config.APP.API_PATH + '/<%= resource %>',
```

Comment:
Yeah I'd like to keep the `url` in the blueprint, maybe we add a not the the default url is a computed property using config values and to remove to use default.